src/api/transactionApi.ts

```diff
@@ -0,0 +1,40 @@
+import {TransactionType} from '../models';
+import {RootStore} from '../stores';
+import {HistoryApi, RestApi} from './index';
+import {ApiResponse} from './types/index';
+
+export interface TransactionApi {
+  fetchWalletTransactions: (
+    walletId: string,
+    skip: number,
+    take: number,
+    assetId?: string,
+    operationType?: TransactionType[]
+  ) => ApiResponse<any>;
+}
+
+export class RestTransactionApi extends RestApi implements TransactionApi {
+  private historyApi: HistoryApi;
+
+  constructor(rootStore: RootStore) {
+    super(rootStore);
+    this.historyApi = new HistoryApi(rootStore);
```

Comment:
probably it's better to instantiate HistoryApi when passing it as a param? kinda DI way